examples/get_imc_data.py

```diff
@@ -0,0 +1,172 @@
+import json
+import os
+from typing import Mapping, Tuple, Union
+
+import click
+import numpy as np
+from skimage.io import imread
+from slicedimage import ImageFormat
+
+from starfish.experiment.builder import FetchedTile, TileFetcher, write_experiment_json
+from starfish.types import Coordinates, Features, Indices, Number
+
+
+class ImagingMassCytometryTile(FetchedTile):
+
+    def __init__(self, file_path: str) -> None:
+        """Initialize a TileFetcher for Imaging Mass Cytometry Data"""
+        self.file_path = file_path
+        self._tile_data = imread(self.file_path)
+
+    @property
+    def shape(self) -> Tuple[int, ...]:
+        return self._tile_data.shape
+
+    @property
+    def coordinates(self) -> Mapping[Union[str, Coordinates], Union[Number, Tuple[Number, Number]]]:
+        # TODO ambrosejcarr: ask about what these coordinates should correspond to.
```

Comment:
going to defer to the wisdom of the person who made [this comment](https://github.com/spacetx/starfish/pull/647#discussion_r220971185) :)